README.md

```diff
@@ -61,6 +61,7 @@ Elafir | Made it possible to control gates with redstone
 VoChiDanh | Refactored parts the PersistentData class in an attempt to resolve java compatibility issues
 alyphen | Transitioned the project from using maven to gradle.
 [JackAttack924](https://github.com/jackcayc924) | Created Tab-Completion functionality for commands
+ConfusedAlex | Added null check for gate coordinates and mob protection flag
```

Comment:
Could you add a link to my GitHub Profile?

Reply:
Yup!

Reply:
Added!